fmskill/compare.py

```diff
@@ -283,12 +289,12 @@ def skill(
         end : (str, datetime), optional
             end time of comparison, by default None
         area : list(float), optional
-            bbox coordinates [x0, y0, x1, y1], 
-            or polygon coordinates [x0, y0, x1, y1, ..., xn, yn], 
+            bbox coordinates [x0, y0, x1, y1],
+            or polygon coordinates [x0, y0, x1, y1, ..., xn, yn],
             by default None
         df : pd.dataframe, optional
```

Comment:
Add an example of motivating the use `df` argument, it is not obvious why this is necessary.

Reply:
Yes, it's only for advanced usage as explained in #33 - I will add example